src/com/hms/flexyosisoftconnector/OsisoftJsonPayload.java

```diff
@@ -346,7 +347,28 @@ public void completePayloadAttempt() {
             firstPayload = false;
           }
 
-          payload.append(PayloadBuilder.addContainerStartToOMFDataMessage(tagName));
+          String typeString = "";
+
+          if (tagPayload.getDataPoint().getType() == DataType.FLOAT) {
+            typeString = "number";
+          } else if (tagPayload.getDataPoint().getType() == DataType.INTEGER) {
+            typeString = "integer";
+          } else if (tagPayload.getDataPoint().getType() == DataType.BOOLEAN) {
+            typeString = "boolean";
+          } else if (tagPayload.getDataPoint().getType() == DataType.DWORD) {
+            typeString = "number";
+          } else if (tagPayload.getDataPoint().getType() == DataType.STRING) {
+            typeString = "string";
+          } else {
+            Logger.LOG_SERIOUS(
+                "Unsupported tag type was selected for tag "
+                    + tagPayload.getDataPoint().getTagName());
+            Logger.LOG_SERIOUS(
+                "Please change that tag type to one of the following: Integer, Boolean, DWord,"
```

Comment:
This log statement should clarify that it is the Ewon tag type, not the OSIsoft tag.